Cargo.toml

```diff
@@ -76,10 +76,10 @@ tt = { path = "./crates/tt", version = "0.0.0" }
 vfs-notify = { path = "./crates/vfs-notify", version = "0.0.0" }
 vfs = { path = "./crates/vfs", version = "0.0.0" }
 
-# In-tree crates that are published separately and follow semver.
-line-index = { version = "0.1.0-pre.1", path = "lib/line-index" }
-la-arena = { version = "0.3.1", path = "lib/la-arena" }
-lsp-server = { version = "0.7.1", path = "lib/lsp-server" }
+# In-tree crates that are published separately and follow semver. See lib/README.md
+line-index = { version = "0.1.0-pre.1" }
+la-arena = { version = "0.3.1" }
+lsp-server = { version = "0.7.1" }
```

Comment:
Up above is a `[patch.'crates-io']` section with a lot of commented out patches, could you  add commented out path versions for these there? That would make it easier to use the local ones on demand for testing.

Reply:
Good idea. Done. I also amended the last bit that I'd added to libs/README.md accordingly.